scripts/variantstore/README.md

```diff
@@ -0,0 +1,131 @@
+  NgsFilterExtract.output_file_base_name | Base name for temporary files in the dataset (DEVNOTE: should make optional) |
+
+  
+## Extracting Cohort
+
+Extracting a cohort currently consists of two steps
+
+1. Prepare - A python script to transform the data in BigQuery into a staging table for extract by the WDL
+2. Extract - A WDL that scatters and extracts the cohort into sharded VCFs
+
+Eventually this python script will be turned into it's own WDL or incorporated into the extract WDL.
+
+### Prepare
+
+The python script is located in `https://github.com/broadinstitute/gatk/tree/ah_var_store/scripts/variantstore/wdl/extract/ngs_cohort_extract.py`
+
+This script requires the following parameters:
+
+| Parameter Name| Description |
+| ----------------------- | ------------- |
+| fq_petvet_dataset       | Dataset containing the data to be extracted|
+| fq_temp_table_dataset   | Dataset for temp tables, described in prereqs  |
+| fq_destination_dataset  | Dataset for prepared cohort, can be same as temp table dataset |
+| destination_table       | Unique name for the destination extract table |
+| fq_cohort_sample_names  | Table containing list of samples to be extracted.  Has a single column `sample_id` with the numeric id of the each sample from the sample map |
+| min_variant_samples     | (Optional) minimum number of variant samples at each site to be extracted |
+| query_project           | Google project where queries are executed, for billing purposes |
+| fq_sample_mapping_table | Complete sample mapping table, typically the `metadata` table in the dataset |
+
+
+An example invocation is as follows
+```
+PROJECT="my-example-project"
+DATASET="bq_wgs_example"
+
+python ngs_cohort_extract.py \
+  --fq_petvet_dataset ${PROJECT}.${DATASET} \
+  --fq_temp_table_dataset ${PROJECT}.temp_tables \
+  --fq_destination_dataset ${PROJECT}.${DATASET} \
+  --destination_table exported_cohort_100_test \
+  --fq_cohort_sample_names ${PROJECT}.${DATASET}.cohort_10_of_35 \
+  --min_variant_samples 0 \
+  --query_project ${PROJECT} \
+  --fq_sample_mapping_table ${PROJECT}.${DATASET}.metadata
+```
+
+
+
+### Extract
+
+Once the prepare step has been completed, the `ngs_cohort_extract.wdl` WDL.  An example inputs json for this workflow can be found at `ngs_cohort_extract.inputs.json`
+
+Most of the parameters in the WDL are constant, such as the reference genome.  However, several should be changed for each run.  These are:
+
+| Parameter Name| Description |
+| ------------- | ------------- |
+| NgsCohortExtract.fq_sample_table | Table of samples to be extracted, same as `fq_cohort_sample_names` from prepare
+| NgsCohortExtract.fq_cohort_extract_table | Fully qualified name of the extract table, comprised of the `fq_destination_dataset` and `destination_table` from the prepare script|
+| NgsCohortExtract.fq_filter_set_table | Name of the filter set table to use, from the filter model training |
+| NgsCohortExtract.filter_set_name | Name of the filter set |
```

Comment:
I'd thought that retraining the filtering was incremental. Let me ask about an example:

- Say at time t1, I have 20K samples, and I train the filter: filter_set_1
- then, at time t2, I have 30K samples, and I train the filter again: filter_set_2

Would you expect filter_set_2 to subsume filter_set_1 for extraction purposes after time t2, and when running the training, we could optionally take advantage of the fact that we'd already built filter_set_1 (and presumably build on it)?